my_list.rb

```diff
@@ -0,0 +1,12 @@
+# This class represents a custom list and includes MyEnumerable module.
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&)
+    # Implementation of each method
+  end
+end
```

Comment:
- [ ] Kindly complete each method as it's requested in the requirements

Reply:
done